src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -292,10 +296,14 @@
                        name="{{ feature.key }}"
                        formControlName="{{ feature.key }}">
                 <label class="custom-control-label"
-                       for="{{ feature.key }}">{{ feature.desc }}</label>
-                <cd-helper *ngIf="feature.helperHtml"
-                           html="{{ feature.helperHtml }}">
-                </cd-helper>
+                       for="{{ feature.key }}">{{ feature.desc }}</label><br>
+                <span *ngIf="feature.helperHtml && rbdForm.getValue(feature.key) === false"
+                      class="text-warning">
```

Comment:
we are all about following patterns now right? so let's be strict on that. This is what @afreen23 has done for the rgw form on situations like this
![image](https://github.com/ceph/ceph/assets/71764184/66b0c77b-2079-4052-88fe-738f3b8ea0bb)

Reply:
Just to not confuse, you can always look into carbon guidelines, we are following those only.